prow/tide/search.go

```diff
@@ -27,121 +27,50 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-type searchExecutor func(start, end time.Time) ([]PullRequest, int /*true match count*/, error)
+type querier func(context.Context, interface{}, map[string]interface{}) error
```

Comment:
Can we get some names for these parameters please?

Reply:
Done